contracts/crowdsale/price/CapStagedCrowdsale.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^ 0.4.24;
+
+import "../Crowdsale.sol";
+import "../../math/SafeMath.sol";
+import "../../ownership/Ownable.sol";
+
```

Comment:
two spaces here: run `npm run lint:sol` to see linting errors

Reply:
I've corrected this issue.